### ISSUE TITLE
Not recognizing .bai extensions for index files

### ISSUE THREAD

**Issue by jamesemery** [HadoopGenomics/Hadoop-BAM#112]
I have noticed that when I try to access bam files in an HDFS using the gatk with an index file in the same directory using the ".bai" extension the gatk fails to find the index. However, if the index file has the extension ".bam.bai" it will correctly identify the index.

**Comment by tomwhite** [comment]
Thanks for the report @jamesemery. How are you accessing the files, and what is the error that you get?

**Comment by jamesemery** [comment]
I am running the BQSR on a dataproc cluster using a bam file stored in hdfs:// directory there. Before running with the .bai file the command line for the program would print out the following message: 

`WARNING: no BAM index file found, splits will not be filtered, which may be very inefficient: hdfs://datproc-cluster-4-m:8020/user/emeryj/Solexa-272222.bam.bai`

When I moved the accompanying Solexa-272222.bai file into the same directory it still printed out the warning. When I renamed Solexa-272222.bai to Solexa-272222.bam.bai it no longer printed out the warning.
